Client/Frontend/Library/Downloads/DownloadsPanel.swift

```diff
@@ -32,10 +32,7 @@ class DownloadsPanel: UIViewController,
 
     private lazy var emptyStateOverlayView: UIView = self.createEmptyStateOverlayView()
 
-    lazy var tableView: UITableView = .build { [weak self] tableView in
-        guard let self = self else { return }
-        tableView.delegate = self
-        tableView.dataSource = self
+    var tableView: UITableView = .build { tableView in
         tableView.register(TwoLineImageOverlayCell.self,
```

Comment:
This is strage one. While implementing unit tests, I had a crash here. It seems `self` is accessed on the `delegate` and `dataSource` because of the `deinit` where they're nilled. I'm not 100% how lazy and .build method play here so I'm open to suggestions.